docs/installation guides/Guide d'installation.md

```diff
@@ -0,0 +1,179 @@
+
+![strat_secu_local.png](../captures/strat_secu_local.png)
+
+## Etape 3 : Installer Virtualbox
+
+- Installer la dernière version stable pour Windows de Virtualbox : `winget install Virtualbox` (dans une console ouverte en mode administrateur)
+- Redémarrer le poste
+
+## Etape 4 : Installer Vagrant
+
+- Installer la dernière version stable pour Windows de Vagrant : `winget install Vagrant` (dans une console ouverte en mode administrateur)
+- Ouvrez un autre terminal pour lancer les commandes suivantes (sinon, vagrant n'est pas connu après la première install)
+- Installer les plugins Vagrant suivants :
+  - ```vagrant plugin install vagrant-certificates```
+  - ```vagrant plugin install vagrant-disksize```
+
+Dans le cas de cette erreur : `ERROR:  You must add /C=US/O=Starfield Technologies, Inc./OU=Starfield Class 2 Certification Authority to your local trusted store
+Vagrant failed to load a configured plugin source.`: Utilisez la commande 
+```
+set SSL_CERT_FILE=C:\HashiCorp\Vagrant\embedded\cacert.pem
+```
+## Etape 5 : Installer Git for Windows
+
+- Lancer la commande `winget install Git.Git` (dans une console ouverte en mode administrateur tant que le ticket : https://github.com/mutagen-io/mutagen/issues/491 n'est pas résolu)
+
+## Etape 6 : Configurer Git
+
+- Exécuter les commandes suivantes pour votre configuration globale Git, **dans un git bash**, pas forcément administrateur. Adaptez évidemment les noms...
+
+```
+git config --global --replace-all user.name "First Last"
+git config --global --replace-all user.email "first.last@mail.com"
+git config --global --replace-all pull.rebase true
+git config --global --replace-all core.autocrlf false
+git config --global --replace-all core.eol lf
+git config --global --replace-all core.filemode false
+git config --global --replace-all http.postBuffer 1000000000
+git config --global --replace-all lfs.activitytimeout 60
+git config --global --replace-all http.version HTTP/1.1
+```
+- Pour vérifier votre configuration Git globale : ```git config --list```
+
+## Etape 7 : Installer Docker Devbox Vagrant
+
+- Utiliser un terminal en étant placé dans ```c:\workspace```
+- Cloner le repo github public du projet ```git clone https://github.com/gfi-centre-ouest/docker-devbox-vagrant docker-devbox```
+- Se placer dans le répertoire ```docker-devbox```
+- Copier-coller le fichier ```config.example.yaml``` en le renommant ```config.yaml``` et adapter les valeurs en fonction du besoin.
+Dans `env` vont se trouver des variables utilisées dans les scripts shell de provision de votre VM.
+Le tableau `provision_options` va servir à appeler des scripts supplémentaires présents dans `/provision/options`
+```
+env:
+  AZ_USERNAME: username Azure
+  AZ_PASSWORD: password Azure
+  AZ_TENANT: tenant Azure
+provision_options: ['mon-script']
```

Comment:
`'mon_script'` is not a proper name, it's just one of the predefined scripts in the folder.